gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -166,6 +168,50 @@ def __ge__(self, other):
 
     return other <= self
 
+  def to_vcf_line(self):
+    """Converts the variant back to a line that can be written to a .vcf."""
+    info_list = []
+    for k, v in self.info.iteritems():
+      if v.field_count == '1':
+        info_list.append('%s=%s' % (k, str(v.data)))
+      else:
+        info_list.append('%s=%s' % (k, ','.join([str(x) for x in v.data])))
+
+    info_field = ';'.join(info_list)
+    format_keys = ['GT']
+    if self.calls:
+      format_keys.extend([k for k in sorted(self.calls[0].info)])
+    format_field = ':'.join(format_keys)
+
+    calls_values = []
+    for call in self.calls:
+      genotype_field = MISSING_FIELD_VALUE
+      if call.genotype != [MISSING_GENOTYPE_VALUE]:
+        phase_char = '/' if call.phaseset is None else '|'
```

Comment:
This will need to be updated to include phaseset in info if it is set and not DEFAULT_PHASESET value.